pkg/cmd/prebuild/add.go

```diff
@@ -134,8 +152,11 @@ var prebuildAddCmd = &cobra.Command{
 	},
 }
 
-var runOnAddFlag bool
-
 func init() {
 	prebuildAddCmd.Flags().BoolVar(&runOnAddFlag, "run", false, "Run the prebuild once after adding it")
+	prebuildAddCmd.Flags().StringVar(&branchFlag, "branch", "", "Git branch for the prebuild")
```

Comment:
From what I can see, these flags are not used at all in the command above.

Reply:
Modified many things in the updated code